templates/ts-apollo-mongodb-runtime-backend/.graphqlrc.yml

```diff
@@ -0,0 +1,24 @@
+## GraphQL Config Generated by Graphback
+## Please review configuration and adjust it for your own project
+schema: ./src/schema/*.graphql
+documents: ./client/src/graphql/**/*.graphql
```

Comment:
This might not be required as there is no client queries.
Generally what is the point of having this config file for runtime example? 
All parameters can be setup remotely. For postgress we needed it for migration purposes.

Reply:
Yeah forgot about this.